interop-testing/src/main/proto/io/grpc/testing/integration/test.proto

```diff
@@ -70,6 +70,12 @@ service TestService {
   // first request.
   rpc HalfDuplexCall(stream StreamingOutputCallRequest)
       returns (stream StreamingOutputCallResponse);
+
+  // A new RPC method added to the proto used for testing extensibility.
```

Comment:
This should probably be deleted, as this proto needs to remain in sync with others and it only helps once (before you regen codegen).